editor/editor_node.cpp

```diff
@@ -2280,7 +2280,7 @@ void EditorNode::_edit_current(bool p_skip_foreign) {
 		if (main_plugin) {
 			// special case if use of external editor is true
 			Resource *current_res = Object::cast_to<Resource>(current_obj);
-			if (main_plugin->get_name() == "Script" && current_obj->get_class_name() != StringName("VisualScript") && current_res && !current_res->is_built_in() && (bool(EditorSettings::get_singleton()->get("text_editor/external/use_external_editor")) || overrides_external_editor(current_obj))) {
+			if (main_plugin->get_name() == "Script" && current_obj->get_class_name() != SNAME("VisualScript") && current_res && !current_res->is_built_in() && (bool(EditorSettings::get_singleton()->get("text_editor/external/use_external_editor")) || overrides_external_editor(current_obj))) {
```

Comment:
Could use `Object::cast_to` but then I'd have to either forward declare `VisualScript` or include it, and that felt overkill for this hack.

This should be refactored so that the editor doesn't need to special case `VisualScript` (typically by adding a `ScriptLanguage` API to find if a language can be edited in an external editor or not).